topics/introduction/slides/introduction.html

```diff
@@ -111,63 +124,99 @@
-  - collects all datasets produced by tools you run
-  - collects all operations performed on your data
+- Location of all analyses <img style="float: right;" alt="History" src="../images/history.png" />
+  - collects all datasets produced by tools
+  - collects all operations performed on the data
 
-- At the heart of Galaxy’s reproducibility<br />
-  For each dataset, the history tracks:
+- For each dataset (the heart of Galaxy’s reproducibility), the history tracks
   - name, format, size, creation time, datatype-specific metadata
-  - tool id and version, inputs, parameters
+  - tool id, version, inputs, parameters
   - standard output (`stdout`) and error (`stderr`)
   - state (<span style="background-color: grey">waiting</span>, <span style="background-color: yellow">running</span>, <span style="background-color: green">success</span>, <span style="background-color: red">failed</span>)
   - hidden, deleted, purged
 
+- Multiple histories
+  - each history should correspond to a **different analysis**
+  - need a meaningful **name**
+
+
 ???
 
-We say <i>datasets</i> to refer to files as well as databases
+- We say <i>datasets</i> to refer to files as well as databases
+- Purged means permanently deleted
 
-Purged means permanently deleted
 ---
 
 ### Multiple histories
 
 - You can have as many histories as you want
-
   - each history should correspond to a **different analysis**
-
   - and should have a meaningful **name**
 
+.image-75[![](../images/galaxy_interface_multiple_histories.png)]
+
+???
+
+- Give it a good name so you can find it later. I have around a hundred histories and after a month I can't remember what I was doing in some, so a good name is important.
+- You can drag and drop datasets between histories
+
 ---
 
 ### History options menu
 
-- History behavior is controlled by the *History options* (gear icon) <img style="float: right;" alt="History menu" src="../images/galaxy_interface_history_long_menu.png" />
+.pull-left[
+History behavior is controlled by the *History options* (gear icon)
+![History menu](../images/galaxy_interface_history_long_menu.png)
+]
+.pull-right[
+.image-75[![History options gear button](../images/galaxy_interface_history_menu.png)]
+- *Create New* history will **not** make your current history disappear
+- To see all of your histories, use the history switcher
+
+  .image-50[![History Switcher](../images/galaxy_interface_history_switch.png)]
 
-.image-25[![History options gear button](../images/galaxy_interface_history_menu.png)]
+- *Copy Datasets* from one history to another and save disk space for your quota
+]
 
-- *Create New* history will **not** make your current history disappear
-- To list all your histories, choose *Saved Histories*
-- You can *Copy Datasets* from one history to another
-  - saves disk space and your quota
+???
+
+- Copying datasets between histories does not affect your quota, it doesn't
+  duplicate the dataset, just makes a link.
```

Comment:
- Copying datasets between histories does not affect your quota, only a single copy of the file is stored on disk because datasets are never modified after creation.